Assets/Scripts/Gamestate/AIManager.cs

```diff
@@ -71,6 +71,9 @@ private void Start()
         aiTargetCollider = Instantiate(aiTarget).GetComponent<AITarget>();
         aiTargetCollider.Owner = this;
         aiTargetCollider.transform.position = transform.position;
+        // Only look for FPS targets in non-auction settings
+        if (FindAnyObjectByType<AuctionDriver>())
```

Comment:
Could be determined by checking for matchmanager as well (it is null when in auction)
...
or, hold on, AuctionDriver is a singleton too, so couldn't you just check that one for nullness?

Reply:
Now checks singleton instead